pages/iam/concepts.mdx

```diff
@@ -54,27 +54,29 @@ You are the [Owner](#owner) of the Organization that is created with your Scalew
 
-This is achieved by inviting users to be Guests in your account's [Organization](#organization), and creating [policies](#policy) that define in a very fine-grained way exactly what permissions they should have for which [resources](#resource) in which of your [Projects](#project) or across your whole Organization.
+This is achieved by adding members to your [Organization](#organization), and creating [policies](#policy) that define in a very fine-grained way exactly what permissions they should have for which [resources](#resource) in which of your [Projects](#project) or across your whole Organization.
 
-Similarly, you may participate as a Guest in someone else's Organization, where you will have the precise rights that they accord to you using policies.
+Similarly, you may be added as a Member in someone else's Organization, where you will have the precise rights that they accord to you using policies.
 
 You can also create non-human users in your Organization, called [IAM applications](#application), in order to give applications programmatic access to your Scaleway resources.
 
 ## Member
 
-You are a Member when you are added to an Organization by an Owner or user with IAM Manager permissions. Members exist only within the specific Organizations in which they are created. This is one of the methods employed at Scaleway to allow Organizations to have multi-users. Members fulfill the same purpose as Guest, while ensuring the security of the Organization.
+You are a Member when you are added to an Organization by an Owner or user with IAM Manager permissions. Members exist only within the specific Organizations in which they are created. This is one of the methods employed at Scaleway to allow Organizations to have multi-users.
 
-As a Member you are subject to [complying with the security requirements](/iam/how-to/comply-with-sec-requirements-member) in effect in your Organization. [Logging into an Organization as a Member](/iam/how-to/log-in-as-a-member) is also different from [logging in as a Guest or as an Owner](/account/how-to/log-in-to-the-console).
+As a Member you are subject to [complying with the security requirements](/iam/how-to/comply-with-sec-requirements-member) in effect in your Organization. [Logging into an Organization as a Member](/iam/how-to/log-in-as-a-member) is also different from [logging in as an Owner](/account/how-to/log-in-to-the-console).
 
 ## Organization
 
-An Organization is made of one or several [Projects](#project). When you create your Scaleway account, an Organization is automatically created, of which you are the Owner. You can also be added to an existing Organization as a [Guest](#guest) or as a [Member](#member).
+An Organization is made of one or several [Projects](#project). When you create your Scaleway account, an Organization is automatically created, of which you are the Owner. You can also be added to an existing Organization as a [Member](#member).
```

Comment:
Is the first part still necessarily true or can Members now have accounts without having their own auto-created Orga?
If so, maybe "Alternatively, you can be added to an existing Organization as a Member without having your own Organization". ?